step2.md

```diff
@@ -53,9 +53,12 @@ Now you have a third file called `cert.pem`.
 
 6. **Add "keys" to your gitignore**. NEVER PUSH YOUR KEYS TO GITHUB! :scream:
 
-7. Change the `BASE_URL` variable in your `config.env` to `https`, instead of `http`.
+7. Browse through the hapi docs again to find out how to create an HTTPS connection on your server.  
+Hint: `server.connection`
 
-8. Push your latest changes to Github....yes the only thing that you have changed is that you added "keys" to your gitignore. But never forget this step :fire:.
+8. Change the `BASE_URL` variable in your `config.env` to `https`, instead of `http`.
+
+9. Push your latest changes to Github....yes the only thing that you have changed is that you added "keys" to your gitignore. But never forget this step :fire:.
```

Comment:
this one might need to be changed slightly now, since you are doing more than just adding keys to gitignore- I guess this could be confusing.